packages/builder/src/internal/cli/project-creation.ts

```diff
@@ -251,21 +248,21 @@ async function installDependencies(
   });
 }
 
-async function getRecommendedDependenciesInstallationCommand(): Promise<
-  string[]
-> {
+async function npmInstallCmd(): Promise<string[]> {
   const isGlobal =
     getExecutionMode() === ExecutionMode.EXECUTION_MODE_GLOBAL_INSTALLATION;
 
-  if (!isGlobal && (await isYarnProject())) {
-    return ["yarn", "add", "--dev", ...SAMPLE_PROJECT_DEPENDENCIES];
+  if (await isYarnProject()) {
```

Comment:
So suddenly we're coupling to yarn?

Reply:
No, it's still in `if`. I added a support to global installation using `yarn`